conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySetsService.kt

```diff
@@ -293,13 +327,17 @@ class OrganizationMetadataEntitySetsService(
         if (!isFullyInitialized()) {
             return
         }
+        ensureOrganizationMetadataEntitySetIdsFullyInitialized(organizationId)
         val organizationMetadataEntitySetIds = organizationService.getOrganizationMetadataEntitySetIds(organizationId)
 
         val datasetEntity = mutableMapOf<UUID, Set<Any>>(
-                propertyTypes.getValue(PGOID).id to setOf(table.oid),
-                propertyTypes.getValue(ID).id to setOf(table.id.toString()),
-                propertyTypes.getValue(DATASET_NAME).id to setOf(table.name),
-                propertyTypes.getValue(STANDARDIZED).id to setOf(false)
+            propertyTypes.getValue(DATASET_NAME).id to setOf(table.name),
+            propertyTypes.getValue(DESCRIPTION).id to setOf(table.description),
+            propertyTypes.getValue(ID).id to setOf(table.id.toString()),
+            propertyTypes.getValue(ORG_ID).id to setOf(organizationId),
+            propertyTypes.getValue(PG_OID).id to setOf(table.oid),
+            propertyTypes.getValue(STANDARDIZED).id to setOf(false),
+            propertyTypes.getValue(TITLE).id to setOf(table.title)
```

Comment:
is this the same entity mapping as above? if yes we should probably extract this into its own like `mapExternalTableToEntity` method to avoid copy/paste issues, especially while the data model here is still developing (assuming we're sticking with the metadata entity set approach, which I have separate opinions about)

Reply:
and same for all the other entity mappings -- column, property type, entity set, etc